test/test_library.py

```diff
@@ -1394,3 +1394,16 @@ def test_durationtype(self):
         beets.config["format_raw_length"] = True
         assert 61.23 == t.format(61.23)
         assert 3601.23 == t.format(3601.23)
+
+
+class PickleTest(ItemInDBTestCase):
+    def test_pickle_dump(self):
+        """Tries to pickle an item. This tests the __getstate__ method
+        of the Model ABC"""
+        import pickle
+
+        item = self.i
+        # Fetch album
+        item._cached_album
```

Comment:
What's the point of this exactly? Is it an indirect way to ensure that the `item` is in fact associated to a database?

This should probably just be
```suggestion
        item.store()
```
which would raise a `ValueError` if there's no database. You could also consider adding an explicit `assert item._db is not None`.